packages/cozy-client/src/__tests__/store.spec.js

```diff
@@ -12,6 +12,7 @@ import reducer, {
 } from '../store'
 import { QueryDefinition as Q } from '../dsl'
 import { TODO_1, TODO_2, TODO_3, TODO_4 } from './fixtures'
+import { removeConsoleMessage } from './utils'
```

Comment:
This can be done a jest config level. 

In package.json the verbose can be switched to `false`. Maybe we should have jest no log anything and if someone wants to have logs, they can put the `--verbose` flag ?